File: submissions/sm_007_brijesh/week_3/day3/session_2/split_string.js
function split_string(string,char){
var arr = [];
var split_char = "" ;
for ( var i = 0; i < string.length; i++ ) {

    if (string[i] != char) {
        split_char += string[i];
    }else{
        arr.push(split_char);
        split_char = "" ;
    }

    }
  arr.push(split_char);
  return arr;
}
console.log(split_string("masai is marvellous" , "a"))

File: submissions/sm_001_aalind/week_13/day_4/session_1/split_string.py
string = "aalind sharma"
split_string = "a"
arr = []
last_i = 0
str = ""

for i in range(len(string)):
    if string[i] != split_string:
        str += string[i]
    else:
        arr.append(str)
        str = ""

arr.append(str)

print(arr)